src/client/src/index.tsx

```diff
@@ -11,9 +11,28 @@ const SimpleLauncher = lazy(() => import('./apps/SimpleLauncher'))
 
 const urlParams = new URLSearchParams(window.location.search)
 
+function getEnv() {
```

Comment:
This is a utility function that can likely be re-used across the codebase. If there's a utility package somewhere, move it to there.

Reply:
Yep tested with the different values!

I've also changed the name of the function to something more appropriate for its move to the rt-util folder: getEnvironment